cedar-policy-cli/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Changed
+- Input policies for `check-parse` command can be read from standard input.
```

Comment:
We'll probably take a separate look at change logs prior to release, but this change in particular isn't included in the release branch